openwisp-qa-check

```diff
@@ -126,6 +131,7 @@ SKIP_CHECKMIGRATIONS=false
 SKIP_MAKEMIGRATIONS=false
 SKIP_FLAKE8=false
 SKIP_ISORT=false
+SKIP_ReSTCHECK=false
```

Comment:
`SKIP_ReSTCHECK` > `SKIP_RESTCHECK`, if we switch to the other tool, we should call it `SKIP_RESTVIEW`

Reply:
Let's change this to `SKIP_RST`